### PR TITLE
bump dotnet 7

### PR DESCRIPTION
**Pull request by bruno-garcia**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,74 @@
...
+- `http://localhost:5000`
+- `https://localhost:5001`
+- `http://localhost:4200` (default `ng serve` port)
+
+> Note: You might need to see how to [install/trust the ASP.NET Core HTTPS development certificates](https://docs.microsoft.com/en-us/aspnet/core/security/enforcing-ssl?view=aspnetcore-5.0&tabs=visual-studio#trust-the-aspnet-core-https-development-certificate-on-windows-and-macos) (not required but nice to have)
+
+### Database backup for contributors
+
+We host a DB backup with the latest month's download counts. This will help you see some data on and fill the charts when developing.
+The backup can be grabbed here: [https://contrib.nugettrends.com/nuget-trends-contrib.dump](https://contrib.nugettrends.com/nuget-trends-contrib.dump) and is a compressed _postgres_ database.
```
Comment (bruno-garcia) [inline CONTRIBUTING.md:63]: Seems nginx isn't running on that box so it's not serving up the backup. Need to address this (not related to the PR)

Comment (bruno-garcia) [reply CONTRIBUTING.md:63]: Raised a separate issue to fix this: https://github.com/dotnet/nuget-trends/issues/207

**Comment by bruno-garcia** [comment]
So far Web's working at least
image

Will test out queuing, scheduler, logging, migrations, etc.

Front-end work will be on a separate PR

**Review by joaopgrassi — APPROVED** [review]
Nice! Thanks for upgrading it 🚀

**Pull request merged**